modules/andes-core/broker/src/main/java/org/wso2/andes/store/rdbms/RDBMSMessageStoreImpl.java

```diff
@@ -112,11 +114,34 @@ public DurableStoreConnection initializeMessageStore(AndesContextStore contextSt
         this.rdbmsStoreUtils = new RDBMSStoreUtils(connectionProperties);
         
         this.messageCache = (new MessageCacheFactory()).create();
+        initializeQueueMappingCache();
+
         log.info("Message Store initialised");
         return rdbmsConnection;
     }
 
     /**
+     * Method to initialize the queue mapping cache.
+     */
+    private void initializeQueueMappingCache() {
+        queueIDs = CacheBuilder.newBuilder()
+                .concurrencyLevel(5)
+                .maximumSize(1000)
```

Comment:
Made them constants. But I do not think we need a configuration for the maximum size. 

A size of 1MB is enough to store more than 10,000 queue names. By making the size 2MB, we can assure that the message cache will have more than enough space to hold all the queue mappings and 2MB is not that big of a size for the memory.

Do you think we still need a configuration for this?